pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -36,7 +36,7 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/device"
 )
 
-func createDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain, c *ConverterContext, virtioNetProhibited bool) ([]api.Interface, error) {
+func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain, c *ConverterContext, virtioNetProhibited bool, ifacesToHotplug ...v1.Interface) ([]api.Interface, error) {
 	if err := validateNetworksTypes(vmi.Spec.Networks); err != nil {
```

Comment:
This part seems irrelevant to hotplug since the networks are auto generated.
I would say to reuse just the part after it.
But to be honest, it is irrelevant for startup as well, the validating webhook should validate it, not the converter... I would remove it.

Reply:
Given the scope / size / number of comments of this PR can we agree on letting this go and polish things along in a follow-up PR ? 

You wanted me to re-use this method - I didn't, but still complied.

Now you want me to split this method so I can use only the relevant part. 

I feel I'm cleaning up the code base as we go along.

Reply:
ok, let's keep the comment unresolved so we can fix it in a follow-up.